centos.sh

```diff
@@ -198,29 +200,29 @@ generate_config_grub() {
   [ "$1" ] || return
   # we should not need to do anything, as grubby (new-kernel-pkg) should have
   # already generated a grub.conf
-  if [ $IMG_VERSION -lt 70 ] ; then
+  if [ "$IMG_VERSION" -lt 70 ] ; then
     DMAPFILE="$FOLD/hdd/boot/grub/device.map"
   else
     # even though grub2-mkconfig will generate a device.map on the fly, the
     # anaconda installer still creates this
     DMAPFILE="$FOLD/hdd/boot/grub2/device.map"
   fi
-  [ -f $DMAPFILE ] && rm $DMAPFILE
+  [ -f "$DMAPFILE" ] && rm "$DMAPFILE"
 
   local i=0
-  for i in $(seq 1 $COUNT_DRIVES) ; do
+  for i in $(seq 1 "$COUNT_DRIVES") ; do
     local j="$[$i-1]"
     local disk="$(eval echo "\$DRIVE"$i)"
```

Comment:
Than $DRIVE should be refactored to an array. And disk should be upper case (like every bash variable). Can you create for the refactor a ticket?